### ISSUE TITLE
Year slider, toggle, and switch controls are not accessible

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#1030]
They don't show up correctly when viewed using contrasting colors.  Seen in both IE and Edge during a user interview.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/05ea5b7c-c145-4a41-8328-78bc81d236df)

![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/e6d39cde-9a44-47c7-90b9-f594285b1dad)

**Comment by jlouis-onrr** [comment]
@mcharg, seems to be the default behavior for toggles, and sliders in mui when in high contrast mode.  You can see these demos in high contrast.  They have the same issues.  Looks like we are going to the hard work for MUI.  Maybe I can do a PR with their repo once I have fix in place. :smile:

https://material-ui.com/components/slider/#accessibility
